src/scrabble.js

```diff
@@ -1,6 +1,103 @@
+const Pointsystem = {
+  1: ['A' , 'a', 'E' , 'e', 'I', 'i', 'O','o', 'U','u', 'L','l', 'N','n', 'R','r', 'S','s', 'T','t'],
+  2: ['D','d', 'G','g'],
+  3: ['B','b', 'C','c', 'M','m', 'P','p'],
+  4: ['F','f', 'H','h', 'V','v', 'W','w', 'Y','y'],
+  5: ['K','k'],
+  8: ['J','j', 'X','x'],
+  10: ['Q','q', 'Z','z']
+};
+
+const letterScores = {
+  a: 1,
+  b: 3,
+  c: 3,
+  d: 2,
+  e: 1,
+  f: 4,
+  g: 2,
+  h: 4,
+  i: 1,
+  j: 8,
+  k: 5,
+  l: 1,
+  m: 3,
+  n: 1,
+  o: 1,
+  p: 3,
+  q: 10,
+  r: 1,
+  s: 1,
+  t: 1,
+  u: 1,
+  v: 4,
+  w: 4,
+  x: 8,
+  y: 4,
+  z: 10
+}
+
+console.log (letterScores)
+
+// Added lowercases to see if it would work assuming the program was case sensitive
+
 class Scrabble {
   // Write your implementation here
 
+  constructor (word) {
+    this.word = word
+  }
+
+  score() {
+
+    // const Totalscore 
+    // const Pointsystem = this.word.split('')
+    if (this.word === null) {
+      return 0
+    }
+
+    
+    // Each individual letter (charAt) in the word is given point and added
+    // Return 0 if nothing is run
+    let totalpoints = 0
+    for (var i = 0; i < this.word.length; i++) {
+      totalpoints += letterScores[this.word.toLowerCase().charAt(i)] || 0;
```

Comment:
Nice and concise, we clearly understand how this system works, if the `letterScores` array returns `undefined` than we ad `0` to the score.